lib/generators/ahoy/stores/templates/intercom_initializer.rb

```diff
@@ -0,0 +1,5 @@
+class Ahoy::Store < Ahoy::Intercom::Store
+  def credentials
+    { token: ENV['INTERCOM_ACCESS_TOKEN']}
```

Comment:
As in the example [here](https://github.com/ankane/ahoy/blob/master/lib/generators/ahoy/stores/templates/kinesis_firehose_initializer.rb)